### PR TITLE
CP-15132: Package daemons for systemd

### PR DESCRIPTION
**Pull request by simonjbeaumont**

### CODE REVIEW

File: SOURCES/xcp-rrdd.service [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=XCP RRD daemon
+After=forkexecd.service xenstored.socket message-switch.service
+Wants=forkexecd.service xenstored.socket message-switch.service
+
+[Service]
+Type=forking
```
Comment (johnelse) [inline SOURCES/xcp-rrdd.service]: Could be `Type=simple` ?

File: SOURCES/xcp-networkd.service [hunk 2]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=XCP networking daemon
+Documentation=man:xcp-networkd(1)
+After=network-online.target forkexecd.service message-switch.service
+Wants=network-online.target forkexecd.service message-switch.service
+
+[Service]
+Type=forking
```
Comment (johnelse) [inline SOURCES/xcp-networkd.service]: `Type=simple`

File: SOURCES/squeezed.service [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Manages host memory by ballooning VMs
+Description=XCP networking daemon
+After=xenstored.socket message-switch.service
+Wants=xenstored.socket message-switch.service
+
+[Service]
+Type=forking
```
Comment (johnelse) [inline SOURCES/squeezed.service]: `Type=simple`

File: SPECS/xenopsd.spec [hunk 4]
```diff
@@ -73,64 +75,68 @@ make
...
+%{__install} -D -m 0755 %{SOURCE1} %{buildroot}%{_unitdir}/xenopsd-xc.service
+%{__install} -D -m 0755 %{SOURCE2} %{buildroot}%{_unitdir}/xenopsd-xenlight.service
+%{__install} -D -m 0755 %{SOURCE3} %{buildroot}%{_unitdir}/xenopsd-simulator.service
+%{__install} -D -m 0644 %{SOURCE4} %{buildroot}%{_sysconfdir}/sysconfig/xenopsd
+%{__install} -D -m 0644 %{SOURCE5} %{buildroot}%{_sysconfdir}/xenopsd.conf
 
 %files
 %doc README.md LICENSE
 %{_libexecdir}/%{name}
 %{_sysconfdir}/xenopsd.conf
```
Comment (johnelse) [inline SPECS/xenopsd.spec]: `xenopsd.conf` is included twice.

**Comment by simonjbeaumont** [comment]
Oops I already had an extra commit to convert these to `Type=simple` systemd services following the investigation into the upgrade testcase. I'll push it here. I'm also gonna push a commit to depend on `syslog.target` where appropriate.

**Comment by simonjbeaumont** [comment]
Because that commit was older than your comments it appears before but you can see it's after the one with the CI tick. Back to you @johnelse...

**Comment by johnelse** [comment]
Looks good!

**Pull request merged**